apps/transaction/forms/transaction_edit_form.py

```
@@ -151,3 +154,8 @@ def _current_total_value(self) -> Decimal:
         if raw_value is None:
             return Decimal("0.00")
         return Decimal(raw_value)
+
+    def _build_category_queryset(self):
```

Ruff (0.14.10)

Missing return type annotation for private function `_build_category_queryset`

(ANN202)